helm-chart/banzai/values-prod.yaml

```diff
@@ -6,20 +6,20 @@
 # will be started when the CPU usage rises above the configured threshold.
 horizontalPodAutoscaler:
   enabled: true
-  minReplicas: 15
-  maxReplicas: 15
+  minReplicas: 25
```

Comment:
We should probably talk about how to introduce autoscaling someday when you have the bandwidth...

Reply:
Do we need to set a max replicas here? Or just assume if we only set a min, they are equal?

Reply:
> We should probably talk about how to introduce autoscaling someday when you have the bandwidth...

Jashan did prototype this at one time. https://github.com/LCOGT/banzai/pull/359/files

Autoscaling based on queue size _sounds_ great Joey, but I am very hesitant to commit my time to this as a development priority. I think it is relatively high effort to low gain, given that we can simply run more pods than we need most of the time.

I would love if someone other than me could spearhead that effort.

Reply:
> Do we need to set a max replicas here? Or just assume if we only set a min, they are equal?

Yes we need to. https://github.com/LCOGT/banzai/pull/359/files

This has always been the case.

Reply:
Yeah. Ok. github split the file and I missed the next line.